baseapp/grpcserver.go

```diff
@@ -23,7 +24,7 @@ import (
 func (app *BaseApp) GRPCQueryRouter() *GRPCQueryRouter { return app.grpcQueryRouter }
 
 // RegisterGRPCServer registers gRPC services directly with the gRPC server.
-func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server) {
+func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server, logQueries bool) {
```

Comment:
nit: I would instead directly supply `grpcConfig` here instead. This future proofs us (even though Baseapp will be sunsetted soon) in case we need other fields or data when calling `RegisterGRPCServer`.